src/reducers/setCursor.js

```diff
@@ -1,134 +1,137 @@
-import * as localForage from 'localforage'
+import * as localForage from 'localforage';
 
 // constants
 import {
-  TUTORIAL_CONTEXT,
-  TUTORIAL_STEP_AUTOEXPAND,
-  TUTORIAL_STEP_AUTOEXPAND_EXPAND,
-  TUTORIAL2_STEP_CONTEXT_VIEW_SELECT,
-} from '../constants.js'
+	TUTORIAL_CONTEXT,
+	TUTORIAL_STEP_AUTOEXPAND,
+	TUTORIAL_STEP_AUTOEXPAND_EXPAND,
+	TUTORIAL2_STEP_CONTEXT_VIEW_SELECT
```

Comment:
The project uses 2 spaces for tabs. Please make sure your editor settings match. If you have the [editorconfig](https://editorconfig.org/) extension in your editor, it will automatically [use these settings](https://github.com/cybersemics/em/blob/dev/.editorconfig).